### PR TITLE
cnf-tests: fix for failing tuning over sriov test

### PR DESCRIPTION
**Pull request by mmirecki**

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/security/tuning_sriov.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package security
 import (
 	"context"
 	"fmt"
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```
Comment (SchSeba) [inline cnf-tests/testsuites/e2esuite/security/tuning_sriov.go:6]: nit: please move the import

File: cnf-tests/testsuites/e2esuite/security/tuning_sriov.go [hunk 2]
```diff
@@ -50,7 +52,18 @@ var _ = Describe("[sriov] Tuning CNI integration", func() {
 			networks.CleanSriov(sriovclient, SriovTestNamespace)
 			sysctls, err := networks.SysctlConfig(map[string]string{fmt.Sprintf(Sysctl, "IFNAME"): "1"})
 			Expect(err).ToNot(HaveOccurred())
-			networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, "test-network", "testresource", sysctls)
+			networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, "test-network", "testresource", fmt.Sprintf("{%s}", sysctls))
```
Comment (SchSeba) [inline cnf-tests/testsuites/e2esuite/security/tuning_sriov.go:55]: general question why we create the network-attachment on the sriov operator namespace?

Comment (mmirecki) [reply cnf-tests/testsuites/e2esuite/security/tuning_sriov.go:55]: Looks like an issue of the original test.
Let's fix it in a follow up PR

**Review by SchSeba — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**